### PR TITLE
Add user-agent to header with sdk version

### PR DESCRIPTION
**Pull request by ManuDeBuck**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     ],
     "require": {
         "php": "^7.0 || ^8.0",
+        "ocramius/package-versions": "^1.9",
```
Comment (hansott) [inline composer.json]:
```suggestion
        "ocramius/package-versions": "^1.4",
```

I guess?

**Review by hansott — COMMENTED** [review]
👌

**Comment by hansott** [comment]
@ManuDeBuck I think we can ignore `ocramius/package-versions`, let's use composer directly: https://getcomposer.org/doc/07-runtime.md

We need to require `composer-runtime-api` as package and then we can use:

```php
\Composer\InstalledVersions::getVersion('journy-io/sdk');
```

And get the version directly 👌 

This should be available for every PHP version we support.

**Comment by ManuDeBuck** [comment]
Thx for the help, @hansott !

**Review by hansott — APPROVED** [review]
🚀

**Pull request merged**